class RetryError(Exception):
    pass


class ServerError(Exception):
    pass
